_dashboards/admin-ui-index/datastream.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Data stream management
+parent: Index and snapshot management in OpenSearch Dashboards
+nav_order: 20
+---
+
+# Data stream management
+Introduced 2.6
+{: .label .label-purple }
+
+In OpenSearch Dashboards, the **Index Management** application allows you to view and manage [data streams]({{site.url}}{{site.baseurl}}/opensearch/data-streams/) as shown in the following image.
+
+![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams1.png)
+
+## Viewing a data stream
+
+To view a data stream and its health status, choose **Data streams** under **Index management** as shown in the following image.
+
+![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams5.png)
+
+The following are the three data stream health statuses:
+
+- Green: All primary and replica shards are assigned.
+- Yellow: At least one replica shard is not assigned.
+- Red: At least one primary shard is not assigned.
+
+## Creating a data stream
+
+To create a data stream, perform the following steps:
+
+1. Under **Index Management**, choose **Data streams**.
+
+1. Choose **Create data stream**.
+
+1. Enter a name for the data stream under **Data stream name**.
+
+1. Ensure that you have a matching index template. This will be populated under **Matching index template**, as shown in the following image.
+
+    ![Data Streams]({{site.url}}{{site.baseurl}}/images/admin-ui-index/datastreams3.png)
+
+1. The **Template details** and **Index alias** sections are read-only. This will inform you of the backing indexes used in the data stream.
+
+1. Optionally, under **Advanced settings** you can specify the number of primary shards, number of replicas, and the refresh interval, as shown in the following image.
```

Comment:
@ariamarble In the create data stream flow, I believe users can't edit any index ailas, settings. They are all read-only and inherited from the matching template.